README.md

```diff
@@ -1,269 +1,81 @@
+const passageConfig = {
+    appID: process.env.PASSAGE_APP_ID,
+    apiKey: process.env.PASSAGE_API_KEY,
 };
 let passage = new Passage(passageConfig);
-
-// example authenticated route
-app.get('/authenticatedRoute', passageAuthMiddleware, async (req, res) => {
-    // get passage user ID from middleware
-    let userID = res.userID;
-
-    // deactivate user
-    let deletedPassageUser = await passage.user.delete(userID);
-    console.log(deletedPassageUser); // true
-});
 ```
 
-## Create A User
-
-You can also create a Passage user by providing an `email` or `phone` (phone number must be a valid E164 phone number).
-
-```javascript
-import { Passage } from '@passageidentity/passage-node';
+### Go Passwordless
 
-let passageConfig = {
-    appID: 'YOUR_APP_ID',
-    apiKey: 'YOUR_API_KEY',
-};
-let passage = new Passage(passageConfig);
+Find all core functions, user management details, and more implementation guidance on our [Passkey Complete Node.js Documentation](https://docs.passage.id/complete/backend-sdks/node) page.
 
-// note that user_metadata is an optional field and is defined in your Passage App settings.
-let newPassageUser1 = passage.user.create({
-    email: 'newEmail@domain.com',
-    user_metadata: {
-        examplefield: 123,
-    },
-});
-console.log(newPassageUser1); // [userObject]
+## Support & Feedback
 
-let newPassageUser2 = passage.user.create({
-    phone: '+15005550006',
-});
-console.log(newPassageUser2); // [userObject]
-```
+We are here to help! Find additional docs, the best ways to get in touch with our team, and more within our [support resources](https://github.com/passageidentity/.github/blob/main/SUPPORT.md).
 
-## Create A Magic Link
+<br />
 
-You can also create a Passage magic link by providing a MagicLinkRequest type
+---
 
-```javascript
-import { Passage } from '@passageidentity/passage-node';
+<p align="center">
+    <picture>
+      <source media="(prefers-color-scheme: dark)" srcset="https://storage.googleapis.com/passage-docs/github-md-assets/passage-by-1password-dark.png">
+      <source media="(prefers-color-scheme: light)" srcset="https://storage.googleapis.com/passage-docs/github-md-assets/passage-by-1password-light.png">
+      <img alt="Passage by 1Password Logo" src="https://storage.googleapis.com/passage-docs/github-md-assets/passage-by-1password-light.png">
+    </picture>
+</p>
 
-let passageConfig = {
-    appID: 'YOUR_APP_ID',
-    apiKey: 'YOUR_API_KEY',
-};
-let passage = new Passage(passageConfig);
-
-let magicLink = passage.createMagicLink({
-    email: 'newEmail@domain.com',
-    channel: 'email',
-});
-```
+<p align="center">
+    <sub>Passage is a product by <a href="https://1password.com/product/passage">1Password</a>, the global leader in access management solutions with nearly 150k business customers.</sub><br />
+    <sub>This project is licensed under the MIT license. See the <a href="LICENSE.md">LICENSE</a> file for more info.</sub>
```

Comment:
This should be just `LICENSE` to make the relative link work within this repo (sorry I didn't catch this in the template!)

Reply:
Oops, I had fixed that but forgot I updated the branch name. Done!